docs/modules/general/proto.mdx

```diff
@@ -0,0 +1,335 @@
+- Added `<match-started/>`, `<match-running/>` & `<match-finished/>` filters that are used to check the state of the current match.
+- New `pre-match-physics` attribute for the terrain module to enable physics events before the match starts.
+
+### Changes in 1.4.1
+
+<span className="badge badge--danger">Breaking</span>
+<br />
+
+<span className="badge badge--secondary">Not implemented</span>
+
+- `<random>` filters now only respond to instantaneous events, and abstain when used in any persistent context. This breaks some uses that relied on undefined behavior, but these would likely break anyway as filters gain more dynamic functionality.
+  ***
+
+<span class="badge badge--primary">Change</span>
+<br />
+
+- <span className="badge badge--secondary">Not implemented</span> As of Minecraft
+  1.9, TNT fuse time is no longer limited to 4 seconds.
+- Portals react to player move events allowing almost instant teleportation as soon as the filter matches.
+  ***
+
+<span class="badge badge--success">New</span>
+<br />
+
+- Added `<observing>` and `<participant>` filters. Also added an `observers` filter property to portals to restrict observer access.
+- Added player rank and score filters. Filters return if the player's rank or score is within the specified range.
+- <span className="badge badge--secondary">Not implemented</span> Added a gliding
+  filter, returns if the player is gliding with an elytra.
+- Control points have new `recovery` and `decay` attributes that replace the `incremental` attribute and allow more control of the progress.
+
+### Changes in 1.4.0
+
+<span className="badge badge--danger">Breaking</span>
+<br />
+
+- Filters, regions, and kits now use `id` instead of `name`.
+  - Keep in mind that IDs are all in the same namespace, so you can not use the same ID for two different types of thing.
+- Teams have a `id` attribute and are always referenced by it everywhere in the XML, **never** by name.
+- Standalone filter definitions are no longer wrapped in a `<filter>` tag, they start with an actual filter, just like regions, e.g.
+
+  - ```xml
+    <filters>
+      <team id="only-red">red-team</team>
+    </filters>
+    ```
+
+- The region for an `<apply>` must be either a `region` attribute or a `<region>` sub-element. It cannot appear directly inside the `<apply>` tag.
+- Inline filters must always be a single tag, multiple tags are not accepted anywhere. To combine multiple filters, always use a compound filter like `<all>` or `<any>`. This change is retroactive, and affects _all_ proto versions.
+- There is no longer a filter type called `<block>`, use `<material>` instead. `<block>` is always interpreted as a region.
+- The old built-in filters are gone, and there are only two new ones: `always` which is equivalent to `allow-all`, and never which is equivalent to `deny-all`.
+  - All of the new built-ins have equivalent tags of the same name i.e. `<always/>` & `<never/>`.
+- Filters can no longer have parents.
```

Comment:
I'm not sure what the old docs mean by this, but filters can be placed inside filters obviously. I would remove this since it's confusing